### ISSUE TITLE
cifar10 and cifar100 parameters

### ISSUE THREAD

**Issue by CandiceUIC** [bhanML/Co-teaching#7]
Hey guys, thanks for your nice work.
I am now trying to re-produce your result presented in the paper. However, I could not reach your result on cifar10 and cifar100. cifar10 is just a bit lower (71.8%  with 45% pairflip noise), cifar100 is much lower than your presented result (31% with 45%pairflip noise). 
In the paper, you mentioned the parameter of batch size, learning rate etc. I am wondering if you have changed settings for different datasets? If so, could please share them?

Cheers
Candice

**Comment by devraj89** [comment]
This is also exactly what I have found.
Also strangely enough when I ran on PyTorch based on Cuda75 I am getting around ~34% but on Cuda80 I am getting around 31%. 
@CandiceUIC Did you manage to find the reason for this ?

Thanks 
Devraj

**Comment by CandiceUIC** [comment]
Cuda75 can reach 34%? That is much exciting. I will try on cuda75.
I don't know exactly what the reason could be for lower performance. But I am thinking about the python/pytorch version. I changed code from python2 to python3, I don't know if that matters. I remembered they presented 34.x in the paper. If so, 34% is close enough I guess.

**Comment by devraj89** [comment]
Hi @CandiceUIC if you require I can send you an yml file where I was getting results close to them.
But can you please let me know if you also observe not a smooth accuracy curve. In the paper, the accuracy of both the models is very smooth but when I ran it it was wildly varying. 
Check https://github.com/bhanML/Co-teaching/issues/3

**Comment by CandiceUIC** [comment]
@devraj89 
I actually got almost similar curve as mentioned in the paper. You should check the number of epoch. Because when I changed epoch number, the curve became not smooth. I guess that is because they have some parameters which are very related to epoch number.
Also, could you please share your file with me? My email address is: 1059113323hy@gmail.com
Cheers
